dags/data_contributions.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import airflow.operators
+from airflow.models import DAG
+import utils.helpers as helpers
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 3, 1),
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag = DAG(
+    dag_id='data_contributions',
+    default_args=args,
+    max_active_runs=1,
+    schedule_interval='@daily'
```

Comment:
This setting is for the DAG itself, not for a specific task. So the DAG would run daily. As the `data_contributions_processor_task` used to depend on the sensor, it would only run when the sensor was successful.

I'm not sure about the relationship between sensors and the `schedule_interval`, as in if the sensor would run following this schedule. I looked around the docs but couldn't find anything, so I asked on Airflow's Gitter. Will let you know if/when someone answers.